.env

```diff
@@ -1,7 +1,6 @@
 EE_IMAGE=quay.io/quay/mirror-registry-ee:latest
 EE_BASE_IMAGE=registry.redhat.io/ansible-automation-platform-22/ee-minimal-rhel8:1.0.0-249
 EE_BUILDER_IMAGE=registry.redhat.io/ansible-automation-platform-22/ansible-builder-rhel8:1.1.0-103
-POSTGRES_IMAGE=registry.redhat.io/rhel8/postgresql-10:1-203.1669834630
-QUAY_IMAGE=registry.redhat.io/quay/quay-rhel8:v3.8.15
+QUAY_IMAGE=quay.io/omr-sqlite/omrrepo:workingsqlite
```

Comment:
It should be a Quay image that we maintain. Ideally from redhat.redhat.io, but a nightly build from quay.io/projectquay/quay would work too.